dip-template/runtimes/dip-sender/src/dip.rs

```diff
@@ -0,0 +1,70 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2023 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use codec::{Decode, Encode};
+use dip_support::VersionedIdentityProofAction;
+use pallet_dip_sender::traits::{
+	DefaultIdentityProofGenerator, DefaultIdentityProvider, TxBuilder, XcmRouterDispatcher,
+};
+use xcm::{latest::MultiLocation, DoubleEncoded};
+
+use crate::{DidIdentifier, Runtime, RuntimeEvent, XcmRouter};
+
+#[derive(Encode, Decode)]
+enum ReceiverParachainCalls {
+	#[codec(index = 50)]
```

Comment:
That's the index of the dip-receiver pallet on the receiver chain. I think we can make this dynamic in the future, but right now the information has to be statically known to compose the `Transact` call.

Reply:
ah so you basically recreate the RuntimeCall enum?

Reply:
Yes, only for the relevant pallet(s). There are talks on how this can be improved, e.g., [here](https://github.com/paritytech/polkadot/issues/5631). But for now this seems to be the only way.